OpenKeychain/src/main/res/values/strings.xml

```diff
@@ -332,6 +334,7 @@
     <string name="public_key_deletetion_confirmation">"Delete key '%s'?"</string>
     <string name="also_export_secret_keys">"Also export secret keys"</string>
     <string name="reinstall_openkeychain">"You encountered a known bug with Android. Please reinstall OpenKeychain if you want to link your contacts with keys."</string>
+    <string name="alert_strip">"Stripping this subkey will render it useless."</string>
```

Comment:
"useless" is not a good word here, maybe "make it unusable on this device!" would be better? adding the "on this device" part, because that's the whole point of it